tools/integration-tests/tester/tests/consensus/consensus_conflicts_test.go

```diff
@@ -92,16 +83,18 @@ func TestConsensus(t *testing.T) {
 		pledgingTxs[receiverId], pledgeSeed[receiverId] = CreateOutputs(pledgeInput, balance, destGenSeed.KeyPair(uint64(receiverId)), 1, peer.ID(), "equal")
 
 		// issue the transaction
-		n.Peers()[0].SendTransaction(pledgingTxs[receiverId].Bytes())
+		_, err = n.Peers()[0].SendTransaction(pledgingTxs[receiverId].Bytes())
+		require.NoError(t, err)
 		receiverId++
+		time.Sleep(2 * time.Second)
 	}
 	// sleep 3* the avg. network delay so both partitions confirm their own pledging transaction
 	// and 1 avg delay more to make sure each node has mana
 	log.Printf("waiting 2 * %d seconds avg. network delay + 5s to make the transactions confirmed", framework.ParaFCoBAverageNetworkDelay)
-	time.Sleep(time.Duration(framework.ParaFCoBAverageNetworkDelay)*2*time.Second + 5*time.Second)
+	//time.Sleep(time.Duration(framework.ParaFCoBAverageNetworkDelay)*2*time.Second + 5*time.Second)
```

Comment:
The Sleep is removed, but the log is still there?

Why has the sleep been moved up? Doing the more sophisticated sleep based on `ParaFCoBAverageNetworkDelay` seems like a good idea...

Reply:
It's commented during debugging. :stuck_out_tongue: